### PR TITLE
Update

### PR DESCRIPTION
**Pull request by RoseBatoeva**

### CODE REVIEW

File: styles/coffee.css [hunk 1]
```diff
@@ -33,4 +33,24 @@
...
+}
+
+@media (min-width: 576px) {
+.coffee-cards-gutter-vertical {
+    background-color: royalblue;
+    width: 6%;
+}
+.coffee-cards-gutter-horizontal {
+    background-color: rosybrown;
+    width: 100%;
```
Comment (trc485) [inline styles/coffee.css:45]: we want the gutter horizontal to assign a space similar to the gutter vertical.
since you used responsive unit (%) there is a trick for this.
gutter vertical: width 6% means 6% of the whole width of the container.
To replicate this in gutter horizontal we need to assign this rule
padding-bottom: 6% or padding-top: 6%
when padding-bottom or top are in percentage, this is calculated form the width of the container.

to summarize. add rule
```css
padding-top: 6%;
```

File: styles/coffee.css [hunk 2]
```diff
@@ -33,4 +33,24 @@
...
+    width: 100%;  
+}
+.coffee-cards {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+}
+.coffee-card {
+    width: 47%;
+    margin: 0;
```
Comment (trc485) [inline styles/coffee.css:54]: margin: 0 is not enough to override the previous rule./
before you said margin-bottom: 30px.
margin-bottom is more specific than margin. So margin will not override margin-bottom.

change this to:
```css
margin-bottom: 0;
```

File: styles/coffee.css [hunk 3]
```diff
@@ -33,4 +33,24 @@
...
+    background-color: rosybrown;
+    width: 100%;  
+}
+.coffee-cards {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+}
+.coffee-card {
+    width: 47%;
```
Comment (trc485) [inline styles/coffee.css:53]: This rule works but to be more precise it should be written like so
```css
flex: 1 1 calc(50% - 6%);
```
because 6% is the width of the gutter vertical
I hope you get what I mean.
And btw check out calc function in css, it is really handy

File: styles/coffee.css [hunk 4]
```diff
@@ -33,4 +33,24 @@
 }
 .coffee-card-description {
     line-height: 1.5;
+}
+
+@media (min-width: 576px) {
+.coffee-cards-gutter-vertical {
+    background-color: royalblue;
```
Comment (trc485) [inline styles/coffee.css:40]: of course, once the page looks good remove these bg colors and the content vert and hor

File: styles/coffee.css [hunk 5]
```diff
@@ -33,4 +33,23 @@
...
+    width: 6%;
+}
+.coffee-cards-gutter-horizontal {
+    width: 100%;  
+    padding-top: 6%; /* when padding-bottom or top are in percentage, this is calculated form the width of the container. */
+}
+.coffee-cards {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
```
Comment (trc485) [inline styles/coffee.css:49]: you need to add
```css
align-items: strech;
```

File: styles/coffee.css [hunk 6]
```diff
@@ -1,36 +1,71 @@
...
+  margin-bottom: 30px;
+  flex: 1 1 calc(50% - 6%);
 }
-
 .coffee-card:last-of-type {
-    margin-bottom: 0;
+  margin-bottom: 0;
 }
-
 .coffee-card-heading {
```
Comment (trc485) [inline styles/coffee.css:22]: please change name of this class to 
coffee-card-header

File: styles/coffee.css [hunk 7]
```diff
@@ -1,36 +1,71 @@
...
-    background-color: #f8f8f8;
-    padding: 15px;  
+  background-color: #f8f8f8;
+  padding-top: 25%;
+  position: relative;
+  flex: 0 0 auto;
+}
+
+.coffee-card-heading h2 {
```
Comment (trc485) [inline styles/coffee.css:29]: please create a class coffee-card-heading and use it instead of this nested selector
